### PR TITLE
use origin-base for base image

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@smarterclayton ptal.

File: images/builder/docker/custom-docker-builder/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# This is the image that executes a Docker build inside Origin. It expects a set of
+# This is image executes a Docker build inside Origin. It expects a set of
```
Comment (smarterclayton) [inline images/builder/docker/custom-docker-builder/Dockerfile]: Typo

File: images/builder/docker/custom-docker-builder/Dockerfile [hunk 2]
```diff
@@ -13,10 +13,12 @@
 #
 # The standard name for this image is openshift/origin-custom-docker-builder
 #
-FROM centos:centos7
-
-RUN yum install -y --enablerepo=centosplus epel-release gettext tar automake make git docker
+# This image is intended for testing purposes, it has the same behavior as
+# the origin-docker-builder image, but does so as a custom image so it can
```
Comment (smarterclayton) [inline images/builder/docker/custom-docker-builder/Dockerfile]: Put this into the lede comment

**Comment by bparees** [comment]
updated.

**Comment by bparees** [comment]
@smarterclayton good?

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**